packages/airnode-node/src/api/index.ts

```diff
@@ -69,34 +69,25 @@ function buildOptions(payload: CallApiPayload): adapter.BuildRequestOptions {
   }
 }
 
-async function signResponseMessage(requestId: string, responseValue: string, config: Config) {
+async function signResponseMessage(requestId: string, data: string, config: Config) {
   const masterHDNode = getMasterHDNode(config);
   const airnodeWallet = ethers.Wallet.fromMnemonic(masterHDNode.mnemonic!.phrase);
 
   return await airnodeWallet.signMessage(
     ethers.utils.arrayify(
-      ethers.utils.keccak256(ethers.utils.solidityPack(['bytes32', 'bytes'], [requestId, responseValue || '0x']))
+      ethers.utils.keccak256(ethers.utils.solidityPack(['bytes32', 'bytes'], [requestId, data || '0x']))
     )
   );
 }
 
-async function signRelayedResponseMessage(
-  requestId: string,
-  timestamp: string,
-  relayerAddress: string,
-  responseValue: string,
-  config: Config
-) {
+async function signDataForBeaconUpdate(templateId: string, timestamp: string, data: string, config: Config) {
```

Comment:
Don't refer to beacons specifically anywhere in the Airnode code, it's just an arbitrary requester contract implementation. This is signing with the template ID so could practically be used for anything.